bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/StreamCombiningIterator.java

```diff
@@ -201,10 +215,31 @@ private void completeStream(boolean addEos) {
     }
   }
 
+  private String parseReadSessionId(String stream) {
+    int streamsIndex = stream.indexOf("/streams");
+    return streamsIndex != -1 ? stream.substring(0, streamsIndex) : stream;
```

Comment:
We should have info logs if the readStream string does not contain "/streams"

Reply:
Done